test/sanity/issue5312-window-events-after-page-reload/test.py

```diff
@@ -0,0 +1,84 @@
+import time
+import os
+import sys
+import subprocess
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common import utils
+
+test_dir = os.path.dirname(os.path.abspath(__file__))
+port = str(utils.free_port())
+server_p = os.path.join(test_dir, "../", "http-server.py")
+server = subprocess.Popen(["python", server_p, port])
+
+tpl = open("index.tpl", "r")
+content = tpl.read().replace("{port}", port)
+tpl.close()
+
+html = open("index.html", "w")
+html.write(content)
+html.close()
+
+chrome_options = Options()
+chrome_options.add_argument("nwapp=" + test_dir)
+
+def click_button(elem_id1, elem_id2, content):
+    time.sleep(1)
```

Comment:
why is sleep() needed here?

Reply:
When I debugging this case on my Windows machine, the page responded slowly, so add this setting, I just change my Linux machine, this sleep() is not necessary, so remove them.

Reply:
Does it imply a race condition? It could lead to unstable results when the system is slow for some reason.

Reply:
Yes

Reply:
Yes